front/app/containers/Admin/projects/all/index.tsx

```diff
@@ -57,11 +53,14 @@ export interface Props {
 
 const AdminProjectsList = memo(({ className }: Props) => {
   const { data: authUser } = useAuthUser();
-  const userIsAdmin = !isNilOrError(authUser) ? isAdmin(authUser) : false;
-  const userIsFolderModerator = !isNilOrError(authUser)
-    ? isProjectFolderModerator(authUser.data)
-    : false;
   const isProjectFoldersEnabled = useFeatureFlag({ name: 'project_folders' });
+  const userIsAdmin = isAdmin(authUser) ?? false;
```

Comment:
```suggestion
  const userIsAdmin = isAdmin(authUser);
```
Is the same (with the current implementation).

Reply:
I need this to be boolean so this is the equivalent of `isAdmin(authUser) ? true : false`.

Reply:
isAdmin returns a boolean (false) user is not an admin, right?

Reply:
Yeah, true! I thought it could be undefined as well, for some reason, but it's not the case.. Thanks!